queen-attack.json

```diff
@@ -67,7 +67,7 @@
       "black_queen": {
         "position": "(1,1)"
       },
-      "expected": false
+      "expected": true
```

Comment:
I believe the coordinates will need updating here if we want this to be true - (2, 4) can't attack (1, 1)